### ISSUE TITLE
Title change/roadmap

### ISSUE THREAD

**Issue by cdknight** [rcsc/captive-portal#1]
I propose that we change this repo to be a monorepo for the entire captive portal deployment. I'd suggest `attendance-captive-portal` as a title or maybe just `captive-portal`. 

A future monorepo would include the current JS portal, along with any deployment files (eg. configuration files for `hostapd`, `dnsmasq`, and/or whatever else we need to get our captive portal working). There's also the idea that we could use NixOS to deploy this, since NixOS is pretty well-suited for embedded, so in that case we'd need to make a Nix derivation for the JavaScript program/some additional configuration stuff for a reverse proxy (even though that's not strictly needed), which I should be able to take care of. 

In addition, we should probably make a roadmap for this program so we can keep track of what we need to do.

**Comment by akhilvanka** [comment]
Yeah, I think that's a great idea and allows for things to be really easy. I'll do what I can right now in terms of changing the name, but any other files that are associated with the portal will definitely be added.

**Comment by cdknight** [comment]
Sounds good. I'll keep this issue open since we can use it to track the roadmap and what else needs to be done in terms of deployment to a captive portal appliance.
